src/operations/bitcoin/send_sats.rs

```diff
@@ -31,3 +36,47 @@ pub async fn create_transaction(
 
     Ok(tx)
 }
+
+pub async fn create_payjoin(
+    invoices: Vec<SatsInvoice>,
+    wallet: &Wallet<AnyDatabase>,
+    fee_rate: Option<FeeRate>,
+    pj_uri: PjUri<'_>, // TODO specify Uri<PayJoinParams>
+) -> Result<Transaction> {
+    let (psbt, details) = {
+        let mut builder = wallet.build_tx();
+        for invoice in invoices {
+            builder.add_recipient(invoice.address.script_pubkey(), invoice.amount);
+        }
+        builder.enable_rbf().fee_rate(fee_rate.unwrap_or_default());
+        builder.finish()?
+    };
+
+    debug!(format!("Request PayJoin transaction: {details:#?}"));
+    debug!("Unsigned Original PSBT:", base64::encode(&serialize(&psbt)));
+    let original_psbt = sign_original_psbt(wallet, psbt).await?;
+    info!("Original PSBT successfully signed");
+
+    // TODO use fee_rate
+    let pj_params = payjoin::sender::Params::non_incentivizing();
+    let (req, ctx) = pj_uri.create_pj_request(original_psbt, pj_params)?;
+    info!("Built PayJoin request");
+    let response = reqwest::Client::new()
+        .post(req.url)
+        .header("Content-Type", "text/plain")
+        .body(reqwest::Body::from(req.body))
+        .send()
+        .await?;
+    info!("Got PayJoin response");
+    let res = response.bytes().await?;
+    info!("Response hex:", &res.to_hex());
+    let payjoin_psbt = ctx.process_response(res.to_vec().as_slice())?;
```

Comment:
We might need to decode the body from utf-8 first. [The reference implementation](https://github.com/Kixunil/payjoin/blob/master/payjoin-client/src/main.rs) uses `reqwest::blocking::Response` which is `Read`. The `reqwest::Response` used here is **not** `Read` so we're figuring out the working reference now. 

```suggestion
    let res = response.text().await?;
    let payjoin_psbt = ctx.process_response(res.as_bytes())?;
```

I think this misuse of `reqwest::Response`  is causing a the same problem in https://github.com/chaincase-app/nolooking/pull/52

Reply:
Okay, keep me informed, I'd love to get this in.

Reply:
This might work too:

```
    let res = response.bytes().await?;
    let payjoin_psbt = ctx.process_response(res.as_ref())?;
```

Reply:
Just tried your first suggestion, I got this:
```
Response: {"errorCode":"original-psbt-rejected","message":"The PSBT should be finalized"}
```
Very strange. I'll push the code I used to test.